e2e/select_test.go

```diff
@@ -218,6 +218,18 @@ func TestSelectCommand(t *testing.T) {
 				outformat:     "json",
 				expectedValue: "id0\n",
 			},
+			{
+				name: "input:json-lines,output:json-lines,all-versions:true",
+				cmd: []string{
+					"select", "json",
+					"--all-versions",
+					"--query", query,
+				},
+				informat:      "json",
+				structure:     "lines",
+				outformat:     "json",
+				expectedValue: "",
+			},
 		},
 		"csv": {
 			{
```

Comment:
We can add tests for `csv` and `parquet` too.

Reply:
> We can add tests for `csv` and `parquet` too.

I added the test cases for CSV and fixed the errors I noticed in the test script. However, it seems that the latest Bitnami MinIO version still doesn't support Parquet files. Therefore, I couldn't run the tests through the `select_test.go` script. Additionally, since the `select` command requires an S3 premium membership, I wasn't able to run the tests myself. Nevertheless, there aren't any edge cases or changes specifically affecting Parquet functionality in my commit. Given that I couldn't run the tests and the chances of my changes affecting Parquet cases are almost negligible, I skipped test function for parquet files.